Dockerfile

```diff
@@ -37,11 +37,14 @@ FROM alpine:3.14
 LABEL license='SPDX-License-Identifier: Apache-2.0' \
   copyright='Copyright (c) 2021: Intel'
 
-COPY --from=builder /go/src/github.com/edgexfoundry/device-rfid-llrp-go/LICENSE /
-COPY --from=builder /go/src/github.com/edgexfoundry/device-rfid-llrp-go/Attribution.txt /
-COPY --from=builder /go/src/github.com/edgexfoundry/device-rfid-llrp-go/cmd /
+RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
+RUN apk add --update --no-cache zeromq dumb-init
+
+COPY --from=builder /app/LICENSE /
+COPY --from=builder /app/Attribution.txt /
+COPY --from=builder /app/cmd /
```

Comment:
This copies too much since `cmd` has source code. Need to copy executable and cmd/res separately.

Reply:
copied just the executable and cmd/res separately